app/workers/send_chase_email_to_referees_worker.rb

```diff
@@ -0,0 +1,13 @@
+class SendChaseEmailToRefereesWorker
+  include Sidekiq::Worker
+
+  def perform
+    GetRefereesToChase.new.perform.each do |reference|
+      begin
+        SendChaseEmail.new.perform(refence: reference)
+      rescue StandardError => e
```

Comment:
What's the reason for rescuing errors here?

Reply:
I'd love to have a better answer than it was on the other workers. I assumed it would help debug if a job failed for whatever reason. I'll remove it